packages/framework/.gitignore

```diff
@@ -2,5 +2,6 @@
 /components
 /vendor
 /composer.lock
-/node_modules
-/assets/js/translations.json
+/assets/node_modules
+/assets/package-lock.json
```

Comment:
What do you think about putting these ignores into the separate .gitignore file into assets folder? the same way as eslintignore file is

Reply:
Yes,we can:)Fied